example/spec/parallel/emsReport/emsIngestDistributionReportSpec.js

```diff
@@ -54,6 +54,12 @@ const collection = { name: 'MOD14A1', version: '006' };
 const collectionId = constructCollectionId(collection.name, collection.version);
 const granuleRegex = '^MOD14A1\\.A[\\d]{7}\\.[\\w]{6}\\.006\\.[\\d]{13}$';
 
+const emsTestConfig = {
+  provider: 'CUMULUS',
+  submitReport: true,
```

Comment:
If `ems_submit_report` is `false`, where did you set it to true?

Reply:
We can add the ems configuration to `example/deployment/sandbox.tfvars`, and set the `ems_submit_report` to `true` in master build.  I'm assuming `cumuls-from-source.tfvars` for master build

Reply:
I had this pass on my own `-tf` stack which had just been created that day. Was it not supposed to pass? My understanding of the test may not be accurate or complete. When would we want `submitReport` to be `false`?

Reply:
Worth noting that although the _test's_ config has `submitReport: true`, the actual deployment's config did not.

Reply:
When `ems_submit_report` is true, the report is sent to EMS, and the ems related configuration (host etc.) is actually used.

Reply:
Right, but what if it is only `true` in the test? Shouldn't `expect(parsed.Key.includes('/sent/')).toBe(true);` fail, because the deployment itself has `submitReport: false`? It doesn't fail currently.

Reply:
This should fail if `submitReport: false`, maybe you can add print statement to see if this line is reached.

Reply:
@jennyhliu I've updated it to pull the deployment's config from the lambda's envs. See if 636e3a5 works for you.